cvpaperchallenge-alumni/limit-lab-homepage
# Enhancement/update header and footer

## Issue URL

close #5

## Change overview

- Added necessary information instead of placeholders.
- Arranged the layout.
- Support responsive design for header/footer.

## How to test

1. Check the ci passes
2. Ensure that the header/footer works fine with multiple display width.

    - Header (for desktop PC)

        ![image](https://github.com/user-attachments/assets/a5078e74-7258-4b29-affc-e58d77b51499)

    - Footer (for desktop PC)

        ![image](https://github.com/user-attachments/assets/330a99cf-f6f4-4345-8a92-e6542aac4e65)

    - Header (for mobile)

        ![image](https://github.com/user-attachments/assets/4762f6a7-6791-429f-9e22-fb1d1999f1ff)


    - Footer (for mobile)

        ![image](https://github.com/user-attachments/assets/26ce10cd-792b-4531-ac8f-985d601a9d5b)


## Note for reviewers

> [!Note]
> Since we have not yet received a high-resolution image, the logo image is left as a placeholder.

> [!Note]
> The images that appear when selecting "Top," "Publications," and "Contact" have been taken from the Visual Atoms paper as a "suitable-looking" marker :)
